frontend/src/components/common/form/SearchToggle.tsx

```diff
@@ -0,0 +1,96 @@
+import { ReactComponent as MapSvg } from 'assets/images/icon-map.svg';
+import { ReactComponent as TableSvg } from 'assets/images/icon-table.svg';
+import React from 'react';
+import { useHistory } from 'react-router';
+import styled from 'styled-components';
+
+import TooltipWrapper from '../TooltipWrapper';
+import { ButtonProps } from '.';
+
+export enum SearchToggleOption {
+  /** The map is the active page */
+  Map = 'map',
+  /** Property List View is the active page */
+  List = 'list',
+}
+
+interface ISearchToggleProps extends ButtonProps {
+  /** set the id of the tool tip use for on hover of the plus buttons */
+  toolId: string;
+  /** Which toggle view is currently active */
+  toggle?: SearchToggleOption;
+}
+
+/**
+ * SearchToggle displaying two buttons which act like a toggle to navigate from the map to the property list view (and vise-versa).
+ * @param param0
+ */
+export const SearchToggle: React.FC<ISearchToggleProps> = ({
+  toggle = SearchToggleOption.Map,
+  toolId,
+  ...props
+}) => {
+  const history = useHistory();
+
+  return (
+    <StyledToggle toggle={toggle}>
```

Comment:
Moved repetitive logic into styled component properties.  Makes the implementation here cleaner.